### PR TITLE
Basic style for the Chatroom 

### PR DESCRIPTION
**Pull request by Uschi760**
Just gave a basic style to the chatroom to make it more usable

### CODE REVIEW

File: app/models/message.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Message < ApplicationRecord
   belongs_to :chatroom
-
   belongs_to :user
 
   validates :content, presence: true,
+
```
Comment (danburck) [inline app/models/message.rb:6]: remove space

Comment (danburck) [inline app/models/message.rb:6]: remoce space

File: app/views/chatrooms/show.html.erb [hunk 2]
```diff
@@ -1,13 +1,15 @@
-
+<div class="form_container">
```
Comment (danburck) [inline app/views/chatrooms/show.html.erb:1]: fix indentation

File: app/views/messages/_message.html.erb [hunk 3]
```diff
@@ -1,6 +1,7 @@
  <div class="message-container" id="message-<%= message.id %>">
       <i class="author">
-        <span><%= message.user.email %></span>
+        <span><%= message.user.username.capitalize %></span></br>
```
Comment (danburck) [inline app/views/messages/_message.html.erb:3]: usernames are usually all lower case. So I would delete the capitalize

Comment (Uschi760) [reply app/views/messages/_message.html.erb:3]: I can do ... but here it wasnt capitalized ...

**Review by danburck — APPROVED** [review]
Looks good! I went over and fixed some indentation.

**Pull request merged**